ssas/service/admin/api.go

```diff
@@ -272,7 +272,7 @@ func updateSystem(w http.ResponseWriter, r *http.Request) {
 	_, err = ssas.UpdateSystem(r.Context(), id, v)
 	if err != nil {
 		logger.Errorf("failed to update system; %s", err)
-		service.JSONError(w, http.StatusBadRequest, http.StatusText(http.StatusBadRequest), "failed to update system")
+		service.JSONError(w, http.StatusNotFound, http.StatusText(http.StatusNotFound), "failed to update system")
```

Comment:
![image](https://github.com/CMSgov/bcda-ssas-app/assets/120701369/de3d6d41-3ab9-4c1f-a9d1-4d960cfe1464)
This error at least partially rolls up to the above image; I think it may be better to return the contextually-relevant http status, since this error appears to have multiple potential options for the correct status

Reply:
eg from the above method alone, I see a case where a 400 or a 404 is appropriate, depending on the error

Reply:
Had a conversation with Lauren. Rolling new conditional logic for custom response handling based on downstream errors have involved extra scope that is outside this specific ticket. Will create a new ticket for the custom error handling outside of malformed requests.